Hotfix for native client makefile

Spotted a link ordering problem with recent GCC (reproduced on GCC 6 from Ubuntu 16.10, but not on GCC 4.9.3):
```g++ -o deepspeech `pkg-config --cflags sox` -L../../tensorflow/tensorflow//bazel-bin/tensorflow -L../../tensorflow/tensorflow//bazel-bin/native_client  client.cc `pkg-config --libs sox` -lkissfft -lc_speech_features -ltensorflow -ldeepspeech 
../../tensorflow/tensorflow//bazel-bin/native_client/libdeepspeech.so: undefined reference to `csf_mfcc'```

--- a/native_client/Makefile
+++ b/native_client/Makefile
@@ -27,7 +27,7 @@ LDFLAGS     := -Wl,-rpath-link,$(RASPBIAN)/lib/arm-linux-gnueabihf -Wl,-rpath-li
 endif
 
 CFLAGS  += -L${TFDIR}/bazel-bin/tensorflow -L${TFDIR}/bazel-bin/native_client $(EXTRA_CFLAGS)
-LDFLAGS += -ltensorflow -lkissfft -lc_speech_features -ldeepspeech $(EXTRA_LDFLAGS)
+LDFLAGS += -ltensorflow -ldeepspeech -lc_speech_features -lkissfft $(EXTRA_LDFLAGS)
 
 default: deepspeech
 

